mteb/tasks/Clustering/MLSUMClusteringP2P.py

```diff
@@ -0,0 +1,60 @@
+import datasets
+import numpy as np
+
+from ...abstasks.AbsTaskClustering import AbsTaskClustering
+
+
+class MLSUMClusteringP2P(AbsTaskClustering):
+    @property
+    def description(self):
+        return {
+            "name": "MLSUMClusteringP2P",
+            "hf_hub_name": "mlsum",
+            "description": (
+                "Clustering of newspaper article contents and titles from MLSUM dataset. Clustering of 10 sets on the newpaper article topics."
+            ),
+            "reference": "https://huggingface.co/datasets/mlsum",
+            "type": "Clustering",
+            "category": "p2p",
+            "eval_splits": ["test"],
+            "eval_langs": ["fr"],
+            "main_score": "v_measure",
+            "revision": "b5d54f8f3b61ae17845046286940f03c6bc79bc7",
+        }
+
+    def load_data(self, **kwargs):
+        """
+        Load dataset from HuggingFace hub and convert it to the standard format.
+        """
+        if self.data_loaded:
+            return
+        self.dataset = datasets.load_dataset(
+            self.description["hf_hub_name"],
+            "fr",
+            revision=self.description.get("revision", None),
+        )
+        self.dataset_transform()
+        self.data_loaded = True
+
+    def create_description(self, example):
+        example["text"] = example["title"] + " " + example["text"]
+        return example
+
+    def dataset_transform(self):
+        """
+        Convert to standard format
+        """
+        self.dataset.pop("train")
+        self.dataset.pop("validation")
```

Comment:
It's not available :/ [https://huggingface.co/datasets/mlsum/blob/main/mlsum.py](https://huggingface.co/datasets/mlsum/blob/main/mlsum.py)
It's already split by lang, or maybe i don't know the tricks ?

Reply:
I've tested locally and it seems to work `datasets.load_dataset('mlsum','fr', split='test')` ? But indeed it still downloads the full datasets and just filter the correct splits afterwards, so not particularly useful

It seems to be a limitation of the datasets lib, but I guess it's still cleaner to have it in the load_dataset method rather than as a processing step ?

Reply:
My bad i hadn't seen it like that.
I make the change :+1:

Reply:
Done :+1: